oxide.go

```diff
@@ -179,25 +192,25 @@ func (d *Driver) Create() error {
 			SshPublicKeys: sshPublicKeyIDs,
 			UserData:      base64.StdEncoding.EncodeToString(userData),
 		},
-	})
+	}
+	instance, err := d.oxideClient.InstanceCreate(context.TODO(), icp)
 	if err != nil {
 		return err
 	}
 
 	d.InstanceID = instance.Id
 	d.BootDiskID = instance.BootDiskId
 
-	networkInterfaces, err := client.InstanceNetworkInterfaceListAllPages(context.TODO(), oxide.InstanceNetworkInterfaceListParams{
+	inilp := oxide.InstanceNetworkInterfaceListParams{
 		Instance: oxide.NameOrId(d.InstanceID),
-	})
+	}
+	networkInterfaces, err := d.oxideClient.InstanceNetworkInterfaceListAllPages(context.TODO(), inilp)
 	if err != nil {
 		return err
 	}
 
-	// TODO: Don't assume the first network interface is the one to use.
-	for _, networkInterface := range networkInterfaces {
-		d.IPAddress = networkInterface.Ip
-		break
+	if len(networkInterfaces) > 1 {
```

Comment:
Should this be >= 1?
at least 1 IPAddress needs to be saved, correct?

Reply:
Nice catch.